Sources/Build/Build.swift

```diff
@@ -0,0 +1,61 @@
+import Foundation
+import Subprocess
+
+@main
+struct Build {
+    static func main() async {
+        do {
+            try await Self.withVersion(in: "Sources/VaporToolbox/Version.swift", as: Self.currentVersion) {
+                _ = try await Subprocess.run(
+                    .path("/usr/bin/env"),
+                    arguments: [
+                        "swift", "build",
```

Comment:
Using `env` to call another command is somewhat obsolete and is only really useful in old platforms that we probably don't need to support anyway since Swift doesn't.
We can let `Subprocess` resolve the Swift executable through PATH.
```suggestion
                    .name("swift"),
                    arguments: [
                        "build",
```